src/AuthContext.tsx

```diff
@@ -172,6 +174,12 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
         logOut()
       } else if (!didFetchTokens.current) {
         didFetchTokens.current = true
+        try {
+          validateState(urlParams)
+        } catch (e: any) {
```

Comment:
This type annotation doesn't provide any info, and might as well be removed? All "caught" objects are `any` in TS.